apps/deploy-web/src/pages/user/api-keys/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import { FormattedDate } from "react-intl";
+import { ApiKeyResponse } from "@akashnetwork/http-sdk";
+import { Button, Popup, Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@akashnetwork/ui/components";
+import { Trash } from "iconoir-react";
+import { NextSeo } from "next-seo";
+import { useSnackbar } from "notistack";
+
+import { CreateApiKeyModal } from "@src/components/api-keys/CreateApiKeyModal";
+import Layout from "@src/components/layout/Layout";
+import { RequiredUserContainer } from "@src/components/user/RequiredUserContainer";
+import { VerifiedPayingCustomerRequiredLink } from "@src/components/user/VerifiedPayingCustomerRequiredLink";
+import { useWallet } from "@src/context/WalletProvider";
+import { useDeleteApiKey, useUserApiKeys } from "@src/queries/useApiKeysQuery";
+import { analyticsService } from "@src/services/analytics/analytics.service";
+
+export default function ApiKeysPage() {
```

Comment:
non-blocking. another general suggestion.

It's a good practice to split components into UI components and data container. I'm pretty sure you know the approach :) I'd suggest to follow it as much as possible. This way it would be easier to test them, maintain and generally understand better

Reply:
Yea of course will split :)